### DISCUSSION TITLE
about how to build a firmware  with  debug

### DISCUSSION THREAD

**Discussion by zwy2025** [EdgeTX/edgetx#5744] [Q&A]
how to enable the debug session

option(TRACE_AUDIO "Traces audio enabled" ON)

if(TRACE_AUDIO)
  add_definitions(-DTRACE_AUDIO)
  set(DEBUG ON)
  set(DEBUG_TRACE_BUFFER ON)
endif()
  but  there is no log print through PB10 and PB11

why?and how to do

**Comment by pfeerick** [comment]
So you added `-DTRACE_AUDIO=ON` to your cmake options when setting up the build? Is AUX1 configured as debug under Radio Settings/SYS -> Hardware (you can also use the USB-VCP if USB-CLI support is available for that target)? What radio / target?

**Reply by zwy2025** [reply]
Thank you very much ！ Now It's ok，AUX1 configured as debug is useful！
